plugins/chat/spec/system/reply_to_message/smoke_spec.rb

```diff
@@ -50,7 +50,7 @@
 
       expect(thread_page.messages).to have_message(text: "user1reply")
       expect(thread_page.messages).to have_message(text: "user2reply")
-      expect(channel_page.message_thread_indicator(original_message)).to have_reply_count(3)
+      expect(channel_page.message_thread_indicator(original_message)).to have_reply_count(2)
```

Comment:
This seems unrelated, but the test was failing.. I think reply count should be 2 here from reading the spec